PatchKit/Classes/Views/ImagesStackView.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  ImagesStackView.swift
+//  abseil
+//
+//  Created by Omar Rasheed on 5/24/20.
+//
+
+import UIKit
+
+class ImagesStackView: UIView {
+    
+    private var imagesStackView = UIStackView()
```

Comment:
can this be let? also seems kind of odd that this var name is the same as class name